CMakeLists.txt

```diff
@@ -1,49 +0,0 @@
-cmake_minimum_required(VERSION 3.18.1)
-project(odr-droid CXX)
-
-set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
-
-# create virtual environment in project root and "pip install conan==1.*"
-set(CMAKE_PREFIX_PATH "${CMAKE_CURRENT_LIST_DIR}/bin")
-# or make sure "which conan" points to a version of 1.x
-
-# TODO: call FIND_PROGRAM(conan) here manually so the location is cached for later
-# TODO: use proper virtual environment
-
-if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
-    message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
-    file(DOWNLOAD "https://raw.githubusercontent.com/conan-io/cmake-conan/0.17.0/conan.cmake"
-            "${CMAKE_BINARY_DIR}/conan.cmake"
-            EXPECTED_HASH SHA256=3bef79da16c2e031dc429e1dac87a08b9226418b300ce004cc125a82687baeef
-            TLS_VERIFY ON)
-endif()
-
-# determine abi
-set(ARCH "${CMAKE_ANDROID_ARCH_ABI}")
-string(REPLACE "armeabi-v7a" "armv7" ARCH "${ARCH}")
-string(REPLACE "arm64-v8a" "armv8" ARCH "${ARCH}")
-
-include(${CMAKE_CURRENT_LIST_DIR}/app/conan.cmake)
-conan_cmake_autodetect(settings_host ARCH ${ARCH})
-# inspired from here https://docs.conan.io/en/latest/integrations/cross_platform/android.html
-configure_file(app/toolchain.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/toolchain.cmake @ONLY)
-conan_cmake_install(PATH_OR_REFERENCE ${CMAKE_CURRENT_LIST_DIR}
-        BUILD missing
-        SETTINGS_HOST ${settings_host} compiler.libcxx=c++_shared
-        PROFILE_BUILD ${CMAKE_CURRENT_LIST_DIR}/conanprofile_build.txt
-        PROFILE_HOST ${CMAKE_CURRENT_LIST_DIR}/conanprofile_host.txt
-        ENV_HOST
-            CONAN_CMAKE_TOOLCHAIN_FILE=${CMAKE_CURRENT_BINARY_DIR}/toolchain.cmake
-            ANDROID_NDK_ROOT=${CMAKE_ANDROID_NDK})
-include(${CMAKE_CURRENT_BINARY_DIR}/conan_paths.cmake)
```

Comment:
all of that is handled by conan now, isn't it?

Reply:
right since we now call conan before cmake we do not need the magic of calling conan inside cmake which is kind of a weird dependency anyways